api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/convolutional/AbstractConv.kt

```diff
@@ -36,26 +65,29 @@ public abstract class AbstractConv(
     protected val biasVariableName: String,
     name: String
 ) : Layer(name) {
-    // weight tensors
+
+    /** Tensor with learnable variables for kernel defined by internal shapes */
     protected lateinit var kernel: Variable<Float>
+
+    /** Tensor with learnable variables for bias defined by internal shapes */
```

Comment:
Hmm, seems not clear with "learnable variables" We could see that this is a variable from the type. Maybe better to rename "Kernel weights" and "Bias weights".
All variable has shapes, it's redundant information

Reply:
fixed